From f1d3c381a6be715bd232078cca9b125377e0001e Mon Sep 17 00:00:00 2001
From: Allen Xu <allen.xu@freescale.com>
Date: Tue, 10 Apr 2012 11:35:31 +0800
Subject: ENGR00177241-1 mx6 close APBH DMA clock when no I/O operation

When there is no NAND I/O operation, close all the reference
clock, include GPMI,BCH and APBH clock.

Signed-off-by: Allen Xu <allen.xu@freescale.com>
---
 arch/arm/mach-mx6/clock.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-mx6/clock.c b/arch/arm/mach-mx6/clock.c
index 43ca85f..2c49585 100644
--- a/arch/arm/mach-mx6/clock.c
+++ b/arch/arm/mach-mx6/clock.c
@@ -5086,7 +5086,7 @@ static struct clk_lookup lookups[] = {
 	_REGISTER_CLOCK(NULL, "caam_clk", caam_clk[0]),
 	_REGISTER_CLOCK(NULL, "asrc_clk", asrc_clk[0]),
 	_REGISTER_CLOCK(NULL, "asrc_serial_clk", asrc_clk[1]),
-	_REGISTER_CLOCK("mxs-dma-apbh",	NULL, apbh_dma_clk),
+	_REGISTER_CLOCK(NULL, "mxs-dma-apbh", apbh_dma_clk),
 	_REGISTER_CLOCK(NULL, "openvg_axi_clk", openvg_axi_clk),
 	_REGISTER_CLOCK(NULL, "gpu3d_clk", gpu3d_core_clk[0]),
 	_REGISTER_CLOCK(NULL, "gpu2d_clk", gpu2d_core_clk[0]),
-- 
1.7.9.5

